devops/scripts/benchmarks/utils/result.py

```diff
@@ -44,6 +51,29 @@ class BenchmarkRun:
     )
     compute_runtime: str = "Unknown"
 
+    @classmethod
+    def safe_from_json(cls, json_data):
```

Comment:
why do we need this? can't we just handle `display_label` being None?

Reply:
Removed after switching to metadata usage